py/ai/h2o/sparkling/H2OConf.py

```diff
@@ -437,6 +437,22 @@ def set_run_as_user(self, user):
         self._jconf.setRunAsUser(user)
         return self
 
+    def setExternalH2ODriverIf(self, iface):
```

Comment:
On purpose I'm changing to camel case as later, I would like to start deprecating the snake case getters and setters on `H2OConf`. After the deprecation is done, we can start generating this configuration class